hardware.inc

```diff
@@ -103,6 +104,13 @@ DEF rSB EQU $FF01
 ; --
 DEF rSC EQU $FF02
 
+DEF SCF_START  EQU %10000000 ;Transfer Start Flag (1=Transfer in progress, or requested)
+DEF SCF_SPEED  EQU %00000010 ;Clock Speed (0=Normal, 1=Fast) ** CGB Mode Only **
+DEF SCF_SOURCE EQU %00000001 ;Shift Clock (0=External Clock, 1=Internal Clock)
+
+DEF SCB_START  EQU $7

```

Comment:
There's no need for those to be hexadecimal, tbh.

Reply:
This is true. I made them hexadecimal to be consistent with the PADB_ set that inspired this pull request, but I'm not sure why they were ever hexadecimal in the first place.